resolver/oneInchmulti.js

```diff
@@ -0,0 +1,89 @@
+import axios from "axios";
+const ChainMap = {
+  Arbitrum: 42161,
+  arb: 42161,
+  Avalanche: 43114,
+  avax: 43114,
+  "Binance Smart Chain": 56,
+  bsc: 56,
+  Ethereum: 1,
+  eth: 1,
+  "Gnosis Chain": 0x64,
+  gnosis: 0x64,
+  "Optimistic Ethereum": 10,
+  optimism: 10,
+  Polygon: 137,
+  polygon: 137,
+};
+export async function ExecuteSwap(
+  _buyToken,
+  _sellToken,
+  _sellAmount,
+  _dsaAddress,
+  _chainName,
+  _slippage
+) {
+  let buyTokenArray = _buyToken.split(",");
+  let sellTokenArray = _sellToken.split(",");
+  let sellAmountArray = _sellAmount.split(",");
+  let dsaAddressArray = _dsaAddress.split(",");
+  let chainNameArray = _chainName.split(",");
+  let slippageArray = _slippage.split(",");
+  for (let i = 0; i < buyTokenArray.length; i++) {
+    let singleswapParams = {
+      fromTokenAddress: sellTokenArray[i],
+      toTokenAddress: buyTokenArray[i],
+      amount: sellAmountArray[i],
+      fromAddress: dsaAddressArray[i],
+      slippage: slippageArray[i],
```

Comment:
If you add `disableEstimate: true,` in here then it is not necessary for the user to have balance.

Reply:
That was very helpful. Thank you